### ISSUE TITLE
Not all locales show up in picker

### ISSUE THREAD

**Issue by AetherUnbound** [WordPress/openverse-frontend#2170]
## Description

It appears that not all translated locales are showing up in the locale picker on openverse.org. For instance, Romanian appears fully translated (https://translate.wordpress.org/projects/meta/openverse/) but does not appear in the dropdown.

@krysal pointed out that the Romanian locale file is present locally but it also doesn't show up in staging either.

## Reproduction

1. Go to openverse.org
2. Click on the drop down
3. Observe that Romanian or română is not in the list of available locales

## Additional context

Originated from a discussion in our [Make WP Slack](https://wordpress.slack.com/archives/C02012JB00N/p1676049829672579).

**Comment by PabloMoratinos** [comment]
Spanish from Spain (Español from España) is also missing, it doesn't apear on the list.

**Comment by dhruvkb** [comment]
There are some more. For example, Dutch `nl` is present but Dutch (Belgium) `nl-be` is missing.
